client/src/featureform/register.py

```diff
@@ -153,6 +189,31 @@ def df_transformation(self,
                           name: str = "",
                           description: str = "",
                           inputs: list = []):
+        """
+        Register a Dataframe transformation source. The local.df_transformation decorator takes the contents
+        of the following function and executes the code it contains at serving time.
+
+        The name of the function is the name of the resulting source.
+
+        The specified inputs are loaded into dataframes that can be accessed using the function parameters.
+
+        **Examples**:
+        ``` py
+        @local.df_transformation(inputs=[("source", "one"), ("source", "two")]) # Sources are added as inputs
+        def average_user_transaction(source_one, source_two):                   # Sources can be manipulated by adding them as params
```

Comment:
this convention may imply that the name is generated from the snake case of the function, may be misleading

Reply:
this clearer?